app/src/main/java/net/aiscope/gdd_app/ui/mask/customview/MaskLayer.kt

```diff
@@ -46,171 +33,226 @@ class MaskLayer(
             this.strokeJoin = Paint.Join.ROUND
             this.strokeCap = Paint.Cap.ROUND
             this.color = color
-            this.alpha = MASK_PAINT_ALPHA
+            this.alpha = (ALPHA_OPAQUE * MASK_PAINT_ALPHA).toInt()
```

Comment:
Hmm, if you keep `MASK_PAINT_ALPHA = 0xCC`, wouldn't this operation become unnecessary? Would there by other side effects?

Reply:
Yep, same result, no side effect. Just trying to make it clearer we have 80% opacity, which is not clear with 0xCC (and makes me thinks the const name should actually be MASK_PAINT_OPACITY XD)

Reply:
Just renamed it to MASK_PAINT_OPACITY - are you ok with it?

Reply:
Sounds good :)

Reply:
Great! Thanks! :-)